src/components/Facility/FacilityList.tsx

```diff
@@ -178,7 +178,7 @@ export const FacilityList = () => {
       <div className="mt-4 gap-2 lg:flex">
         <CountBlock
           text="Total Facilities"
-          count={permittedData ? permittedData.count : 0}
+          count={permittedData ? permittedData.count : null}
```

Comment:
Previously, we were passing 0 as a fallback value, which caused 0 to appear in the advanced filter when there was no data. Now, I'm using null instead of 0 as the fallback to avoid displaying a misleading 0 when the data is unavailable.